### PR TITLE
feat: form elements - File upload documentation

### PR DESCRIPTION
**Pull request by catiarodriguescosta**
- Moved component to a new page
- Added properties and examples for wmndsFileUpload component
- No references to be updated
- Updated File upload params
- Grouped params: error message params, group params (no label, no hint and no attributes)

### CODE REVIEW

File: src/wmnds/components/form-elements/file-upload/_properties.njk.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          "name": "classes",
+          "type": "string",
+          "description": "Classes to add to the error message span tag."
+        }
+      ]
+    },
+    {
+      "name": "formGroup",
+      "type": "Object",
+      "description": "Options for the form-group wrapper See formGroup below.",
```
Comment (houbly) [inline src/wmnds/components/form-elements/file-upload/_properties.njk.json:73]:
```suggestion
      "description": "Options for the form-group wrapper. See formGroup options below.",
```

File: src/wmnds/components/form-elements/file-upload/_properties.njk.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      "arrayOptions": [
+        {
+          "name": "classes",
+          "type": "string",
+          "description": "Classes to add to the form group (e.g. to show error state for the whole group)"
```
Comment (houbly) [inline src/wmnds/components/form-elements/file-upload/_properties.njk.json:78]:
```suggestion
          "description": "Classes to add to the form group"
```
could be a little confusing as the error message already adds the error class to the form group

**Review by houbly — CHANGES_REQUESTED** [review]

File: src/wmnds/components/form-elements/file-upload/_properties.njk.json (hunk 2 above)
Comment (catiarodriguescosta) [reply src/wmnds/components/form-elements/file-upload/_properties.njk.json]: completely right! 👍

**Review by houbly — APPROVED** [review]

**Pull request merged**